staging/src/k8s.io/apiserver/pkg/apis/apiserver/validation/validation_test.go

```diff
@@ -585,6 +585,61 @@ func TestValidateAuthenticationConfiguration(t *testing.T) {
 			},
 			want: "",
 		},
+		{
+			name: "valid authentication configuration that uses verified email via claim validation rule",
+			in: &api.AuthenticationConfiguration{
+				JWT: []api.JWTAuthenticator{
+					{
+						Issuer: api.Issuer{
+							URL:       "https://issuer-url",
+							Audiences: []string{"audience"},
+						},
+						ClaimValidationRules: []api.ClaimValidationRule{
+							{
+								// This expression matches the go code we have in the JWT authenticator
```

Comment:
`claims.?email_verified.orValue(true)` already matches the referenced code for a missing email_verified claim... I don't think this matches that code when email_verified is present and non-boolean